web/api/login.go

```diff
@@ -145,10 +153,31 @@ func (ar *Router) IsLoggedIn() http.HandlerFunc {
 	}
 }
 
+// getTokenPayloadForApp get additional token payload data
+func (ar *Router) getTokenPayloadForApp(app model.AppData, user model.User) (map[string]interface{}, error) {
+	if app.TokenPayloadService() == model.TokenPayloadServiceHttp {
+		// check if we have service cached
+		ps, exists := ar.tokenPayloadServices[app.ID()]
+		if !exists {
```

Comment:
Maybe its better to use early return here?
`if exists { return ps.TokenPayloadForApp(app.ID(), app.Name(), user.ID()) } `